.vscode/settings.json

```diff
@@ -5,8 +5,8 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true, // if developer has eslint extension enabled
-    "source.fixAll.tslint": false // if developer has tslint extension enabled
+    "source.fixAll.eslint": "explicit",
```

Comment:
Could you remove the vhanges to this file please? 🙏

Reply:
Sorry, it was an error. It's fixed now! 👍